tech/languages/python/scipy.md

```diff
@@ -250,7 +250,7 @@ Next select the dropdown _New_ then _Python 3_ to create a new notebook.
 
 ![notebook example](https://jupyter.org/assets/jupyterpreview.png)
 
-Installing python3-notebook also installs numpy, pandas, scipy and IPython.
+Numpy, pandas, scipy and IPython are included with the notebook install.
```

Comment:
I was about to let it be, but since you now start with capital N, please use the names as written on the page: NumPy, Pandas, SciPy and IPython.